tests/test_ad_nnlo.py

```diff
@@ -25,7 +25,7 @@ def test_gamma_2():
     sx = np.append(sx, harmonics.harmonic_S4(2))
     gS2 = ad_nnlo.gamma_singlet_2(2, NF, sx)
     # gluon momentum conservation
-    np.testing.assert_allclose(gS2[0, 1] + gS2[1, 1], 0, atol=9e-2)
+    np.testing.assert_allclose(gS2[0, 1] + gS2[1, 1], 0, atol=4e-3)
```

Comment:
Still `1e-3` it's a bit too much... don't you think so?

Reply:
Hi Ale, I checked all the ad with mathematica and now I'm quite sure they all agree with the fortran code and the tests reach the same precision on the fortran code.

Reply:
The difference is proportional to NF or ( NF^2 ) so if you use NF=2 it's around 1e-4

Reply:
- did you check this contribution specifically?
- otherwise I'm afraid the story from NLO continues https://github.com/N3PDF/eko/blob/cc248184e7555ec75bc073cf4a561d8569246899/tests/test_ad_nlo.py#L18

Reply:
What do you mean? Here CA is encoded inside the numbers...
From mathematica and referring to the fortran code I get:
```
GG1 + QG1 = 0.000676541  (* prop to nf^1 *)
GG2 + QG2 = 0.0000561758 (*prop to nf^2 *)
GG0 = -0.0008998329547011963 (*prop tp nf ^0*)
```

Reply:
I believe that the problem it's not CA, but the term whose coefficient is CA...
What Felix said is that that term it's hard to integrate, the convergence it's numerically difficult, thus he lifted a little bit the constraint. But he lifted `2e-6 -> 4e-5`, instead `4e-3` it's completely out by 2000 times, hard to believe it's just integration error, even if tough to converge.

Reply:
Not "integration error" but evaluation error - otherwise yes!